cardano-api/src/Cardano/Api/Value.hs

```diff
@@ -33,13 +33,19 @@ module Cardano.Api.Value
   , MultiAssetInEra(..)
   ) where
 
-import           Prelude
-
-import           Data.String (IsString)
-import           Data.ByteString (ByteString)
-import           Data.Map.Strict (Map)
-import qualified Data.Map.Strict as Map
+import           Cardano.Prelude
```

Comment:
I'd honestly rather we didn't.

I've been porting each one to the normal prelude as we split them out of the big monolithic module.